torch/_torch_docs.py

```diff
@@ -3361,6 +3361,27 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.ge`.
 """)
 
+add_docstr(torch.gradient,
+           r"""
+gradient(input, *, domain=None, dim=None, edge_order=1) -> List of Tensors
+
+This function is analogous to numpy.gradient with minor changes in arguments.
```

Comment:
"This function is analogous to NumPy's gradient function."

make "NumPy's gradient function" a link to the docs for the gradient function